tensorboard/components/tf_paginated_view/tf-category-paginated-view.html

```diff
@@ -165,9 +178,16 @@
         border-bottom-right-radius: 0 !important;
       }
 
+      [open-button] .expand-arrow {
+        transform: rotateZ(180deg);
+      }
+
       .name {
         display: inline-flex;
-        float: left;
+        padding: 2px 0;
+        white-space: pre;
```

Comment:
forgot: how does it differ from `nowrap`?

Reply:
Sequences of white space are preserved.  E.g. if I type "a    b" (many spaces in the middle) in the search field, I prefer seeing what I typed, instead of "a b" (collapsed to 1 space).

Since .category-name actually holds the text, I think this one is unnecessary.